### ISSUE TITLE
which branch?

### ISSUE THREAD

**Issue by simonedebattisti** [DataAccess2020/CE1_Group_JCA#8]
It seems that the master brand is actually behind...is it abandoned?

is jacopo's the final one? or will pull all its changes back into the master...where the other script and rmd fils will also appear?

by the way on the master, i added a few plots for deepdin the thinking on a country base

(well this questions is just to understand, and to try this issue management too )

**Comment by claudiatang95** [comment]
Master branch is already the latest branch, it's already our final project. You are welcome to upload your analysis but please upload to your branch, otherwise it will confuse us.

**Comment by simonedebattisti** [comment]
ok, i did some..but on jacopo's...will not do anymore , sorry

take a look..show some interesting views

so for the markdown..should i duplicate and do my own? i dont want to mess your final project for sure ;-9

**Comment by claudiatang95** [comment]
Yes, you can create a branch from master and write your markdown report in your branch, then it won't change anything of our project in master branch.
